task6/Working with posts.js

```diff
@@ -0,0 +1,251 @@
+(function(){
+class post {
+	constructor(id, avatar, author, createdAt, description, hashTags=[], photoLink="") {
+		this.id = id;
+		this.avatar = avatar;
+		this.author = author;
+		this.createdAt = createdAt;
+		this.description = description;
+		this.photoLink = photoLink;
+		this.hashTags = hashTags;
+		this.likes=[];
+		this.countOfLikes=0;
```

Comment:
If you have links array, then the amount of likes can be obtained by `this.likes.length`.